src/main/java/uk/gov/hmcts/reform/hmc/model/linkedhearinggroup/LinkHearingDetails.java

```diff
@@ -9,6 +10,7 @@
 
 @Data
 @NoArgsConstructor
+@AllArgsConstructor
```

Comment:
do we need both?

Reply:
Yes. Some tests use the noargs and some use the full args.